lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/TestFailMapperTasks.cs

```diff
@@ -67,6 +68,9 @@ public override void TestFailedMapperOnLocalRuntime()
             Assert.Equal(numTasks, completedTaskCount + failedTaskCount);
             Assert.Equal(0, failedEvaluatorCount);
             Assert.Equal(numTasks, runningTaskCount);
+            
+            // job fails
+            Assert.True(jobFailure > 0);
```

Comment:
In case of job failure, we throw exception with this message, and it shows up three times - twice as the message and once as part of stack trace. I didn't put in 3 so that if we modify the behavior later, we don't have to change the constant